shell/assets/translations/en-us.yaml

```diff
@@ -4318,17 +4318,17 @@ plugins:
     title: Extension support is not enabled
     prompt:
       cant: Automatic installation is not available - required Helm Charts could not be found
-      can: You need to install the Extension Operator
+      can: You need to enable the Extensions feature flag in Global Settings. Once enabled, Rancher will require a restart.
```

Comment:
this should receive the name of the FF and capitalise the resource, so

"You need to enable the `{ uiextension }` Feature Flag in Global Settings. Once ena...."